ui/lib/pki/addon/components/pki-import-pem-bundle.ts

```diff
@@ -47,9 +47,29 @@ export default class PkiImportPemBundle extends Component<Args> {
   @tracked errorBanner = '';
 
   get importedResponse() {
-    // mapping only exists after success
-    // TODO VAULT-14791: handle issuer already exists, but key doesn't -- empty object returned here
-    return this.args.model.mapping;
+    const { mapping, importedIssuers, importedKeys } = this.args.model;
+    if (undefined === mapping) return null;
+
+    const importList =
+      (importedIssuers || []).map((issuer: string) => {
+        const key = mapping[issuer];
+        return { issuer, key };
+      }) || [];
```

Comment:
Shouldn't `(importedIssuers || []).map` always return an array? Do we need to fallback to an empty array here?

Reply:
Oh good catch! Typescript kept complaining that the result could be undefined so I went a little overkill 😂

Reply:
Oh Typescript 😅